api/server/router/network/network_routes.go

```diff
@@ -164,6 +164,13 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 		// return the network. Skipped using isMatchingScope because it is true if the scope
 		// is not set which would be case if the client API v1.30
 		if strings.HasPrefix(nwk.ID, term) || (netconst.SwarmScope == scope) {
+			//If we have a previous match "backend", return it, we need verbose when enabled
```

Comment:
small nit, can you put a space after the // of the comment?
applies to both lines and also the test

Reply:
done, PR updated